tests/test_save_identity.py

```diff
@@ -7,11 +7,13 @@
 
 class TestApi(TestCase):
     def test_save_identity(self):
```

Comment:
Any failure scenarios?

Reply:
test recovered to previous version, because it is out of scope of this branch. Added TODO: test failure scenarios